certbot-nginx/certbot_nginx/configurator.py

```diff
@@ -867,3 +875,18 @@ def install_ssl_options_conf(options_ssl):
     # Check to make sure options-ssl.conf is installed
     if not os.path.isfile(options_ssl):
         shutil.copyfile(constants.MOD_SSL_CONF_SRC, options_ssl)
+        return
+    # there's already a file there. if it exactly matches a previous file hash,
+    # we can update it. otherwise, create a .new and print a warning.
+    digest = crypto_util.sha256sum(options_ssl)
+    if digest in PREVIOUS_SSL_OPTIONS_HASHES: # safe to update
+        shutil.copyfile(constants.MOD_SSL_CONF_SRC, options_ssl)
+    elif digest == CURRENT_SSL_OPTIONS_HASH: # already up to date
+        return
+    else: # not safe to update
```

Comment:
I personally think it's nice that we continue to make noise about the problem until the user either fixes the file or tells us they aren't interested in updates.